.github/workflows/reusable_multi_numa.yml

```diff
@@ -61,11 +60,12 @@ jobs:
         # On RHEL, hwloc version is just a little too low.
         # Skip some tests until we upgrade hwloc and update CMake to properly handle local hwloc installation.
         # TODO: fix issue #560
+        # TODO: add issue for -E umf-init_teardown
```

Comment:
Is this test failure also related to `hwloc` version on `rhel`? If not, please add a short comment.

Reply:
I modified this comment slightly but currently I don't know the reason why it fails only there